server/src/main/java/se/fortnox/reactivewizard/server/RwServer.java

```diff
@@ -193,6 +210,25 @@ static void awaitShutdownDependency(int shutdownTimeoutSeconds) {
         LOG.info("Shutdown dependency completed, continue...");
     }
 
+    static void awaitLoggingShutdown(int shutdownTimeoutSeconds) {
+        if (loggingShutdownRunner == null) {
+            return;
+        }
+
+        LOG.info("Wait for logging to shut down");
```

Comment:
Mabe its even easier to express the wait with reactor?

	Mono.fromRunnable(runnableStuff)
			.timeout(Duration.ofSeconds(3))
			.subscribe();

Reply:
It looks nice when expressed via reactor operations. But I suspect that the initial motivation behind using the bare thread semantics was that we want simple low level means of control during the shutdown procedure. I guess that's not the best moment to rely on the reactor's thread behaviour.

Reply:
There's no timeout anymore, so this code is gone.